### PR TITLE
gh-91058: Add what's new entry for the improvement in commit 7cfbb49fcd4c85f9bab3797302eadf93df490344

### PR DESCRIPTION
**Pull request by pablogsal**
* Issue: gh-91058

### CODE REVIEW

File: Doc/whatsnew/3.12.rst [hunk 1]
```diff
@@ -106,6 +106,16 @@ Improved Error Messages
         ^^^^^^^^^^^^^^^^^^^^^^^
     SyntaxError: Did you mean to use 'from ... import ...' instead?
 
+* :exc:`ImportError` exceptiond raised from failed ``from <module> import
```
Comment (AlexWaygood) [inline Doc/whatsnew/3.12.rst]:
```suggestion
* :exc:`ImportError` exceptions raised from failed ``from <module> import
```

File: Doc/whatsnew/3.12.rst [hunk 2]
```diff
@@ -106,6 +106,16 @@ Improved Error Messages
...
+  <name>`` now include suggestions for the value of ``<name>`` based on the
```
Comment (AlexWaygood) [inline Doc/whatsnew/3.12.rst]:
```suggestion
  <name>`` statements now include suggestions for the value of ``<name>`` based on the
```

**Review by AlexWaygood — COMMENTED** [review]
Two small suggestions

**Comment by pablogsal** [comment]
thanks @AlexWaygood ❤️

**Review by AlexWaygood — APPROVED** [review]
Looks good! (And thank you for the awesome feature, it's going to save so much pain ❤️)

**Pull request merged**